USAGE

```diff
@@ -35,6 +35,10 @@ Which is (NBR_PARCEL) * (NBR_PARCEL-1) / 2. Running connectoflow with a freesurf
 parcellation is A LOT faster than with Brainnetome, which is A LOT faster than
 with Glasser, make sure to test & optimize calls before launching on clusters.
 
+If providing a lesion mask, the nifti should be a uint8 binary image representing
+the voxel that are considered lesions. These lesions should have been taken into account
+when generating the tractogram (to allow tractography within the lesions).
+
```

Comment:
While we are on this, can you add a word on the labels from freesurfer or any parcellation in lines 5-8. To make sure the user realizes that the atlas must contain only non-zero labels of the connectome, i.e. strip out the wm labels, csf labels or any labels that are not cortical/subcortical and desired in the connectome.